src/Uno.Sdk/targets/Uno.Common.Wasm.targets

```diff
@@ -41,6 +41,39 @@
 		</ItemGroup>
 	</Target>
 
+	<Target Name="_UnoGenerateGlobalUsingsMock"
+			BeforeTargets="CoreCompile"
+			DependsOnTargets="_UnoFilterWebSdkGlobalUsings"
+			Condition=" '$(Optimize)' != 'true' AND '$(UnoDisableGenerateGlobalUsingsMock)'!='true' ">
+		<!-- 
+			This target is present to work around global usings that provided by the .NET Web SDK
+			that we cannot remove reliably, particular during Hot Reload. 
+			As a workaround we generate a temporary file that contains the namespaces and private 
+			types that will make global using namespaces valid at compile time.
+		-->
+
+		<PropertyGroup>
+			<_FakeGlobalUsingsFile>$(IntermediateOutputPath)/FakeGlobalUsings.g.cs</_FakeGlobalUsingsFile>
+			<_FakeGlobalUsingsAssemblyName>$(AssemblyName.Replace('.','_'))</_FakeGlobalUsingsAssemblyName>
+		</PropertyGroup>
+
+		<ItemGroup>
+			<_UsingsDedup Include="@(Using)" Name="%(Identity)" KeepDuplicates="false" />
+			<_GlobalUsingMocksLines Include="@(_UsingsDedup->'namespace %(Name) { class __PlaceHolderDoNotUse_$(_FakeGlobalUsingsAssemblyName) { } }')" />
```

Comment:
Since this is a work around for Microsoft.AspNetCore.* can we filter on this for the usings?

Reply:
It's a bit larger than that, as it also includes Microsoft.Extensions